docker.go

```diff
@@ -99,5 +106,27 @@ func loadDockerImage(cli *client.Client, pathToImage string) (string, error) {
 
 	b, err := ioutil.ReadAll(ret.Body)
 
-	return string(b[:]), nil
+	// {"stream":"Loaded image: sles12/mariadb:10.0\n"}
+	var loadResponseBody LoadResponseBody
+	if err := json.Unmarshal(b[:], &loadResponseBody); err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(loadResponseBody.Stream)[14:], nil
+}
+
+// Tags the specified docker image with the supplied tags
+func tagDockerImage(cli *client.Client, image string, tags []string) (error) {
+	for _, tag := range tags {
+		log.Debug("Tagging image: ", image, " with ", tag)
+
+		qualifiedTag := strings.Split(image, ":")[0] + ":" + tag
+
+		err := cli.ImageTag(context.Background(), image, qualifiedTag)
+		if err != nil {
+			return err
```

Comment:
Might be just fine, as tagging an image shouldn't fail under normal conditions, but in case it happens, shouldn't we keep trying with the rest of the tags instead of stopping and returning that failure? I mean, we can still return failure but we can try a best-effort here when tagging.